05-html-css/README.md

```diff
@@ -0,0 +1,18 @@
+## HTML & CSS
+
+1. [Intro to the Web](01-the-web.md)
```

Comment:
Could we add some more organization here? Maybe something like

### HTML
1. link
1. link

### CSS
1. link
1. link

### Layout
1. link
1. link